source/draw/RasterMonochrome.ooc

```diff
@@ -224,4 +224,10 @@ RasterMonochrome: class extends RasterPacked {
 		}
 		result
 	}
+	writePixelGpu: override func (x, y: Int, color: ColorRgba) {
```

Comment:
Its purpose is to emulate how the GPU works for portability when moving code. Otherwise, someone will instantly change how it behaves and break portability.